src/main/java/com/softserve/util/FirebaseTest.java

```diff
@@ -0,0 +1,90 @@
+package com.softserve.util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Date;
+import java.util.Objects;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+import com.google.auth.Credentials;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.BlobInfo;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+
+@RestController
+@RequestMapping("/firebase")
+public class FirebaseTest {
+	
+	private StorageOptions storageOptions;
+	
+	private final String bucketName;
+	private final String projectId;
+	private final String serviceAccountKey;
+	
+	public FirebaseTest(@Value("${firebase.bucket-name}") String bucketName,
+						@Value("${firebase.project-id}") String projectId,
+						@Value("${firebase.service-account-key}") String serviceAccountKey) {
+		this.bucketName = bucketName;
+		this.projectId = projectId;
+		this.serviceAccountKey = serviceAccountKey;
+	}
+	
+	@PostMapping("/upload")
+	public String[] uploadTest(@RequestParam("file") MultipartFile multipartFile) {
+		try {
+			return upload(multipartFile);
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+			return new String[]{"Success"};
+		}
+	}
+
+	private String[] upload(MultipartFile multipartFile) throws IOException {
+        FileInputStream serviceAccount =
+                new FileInputStream(serviceAccountKey);
+        
+        this.storageOptions = StorageOptions.newBuilder()
+				                .setProjectId(projectId)
+				                .setCredentials(GoogleCredentials.fromStream(serviceAccount))
+				                .build();
+        
+        
+        File file_ = convertMultiPartToFile(multipartFile);
```

Comment:
we don't use this kind of variables with an underscore at the end.

use "file" instead of "file_"